Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchableVolume.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Add a NearInteractionTouchableVolume to your scene and configure a touchable volume
+    /// in order to get PointerDown and PointerUp events whenever a PokePointer collides with this volume.
+    /// </summary>
+    public class NearInteractionTouchableVolume : BaseNearInteractionTouchable
+    {
+        public override float DistanceToSurface(Vector3 samplePoint, out Vector3 normal)
+        {
+            if (usesCollider)
```

Comment:
what about if the volume does not use the collider? It would be good in the tooltip or class description to say that this class requires that a collider exists on the component and that the collider is used to compute the touchable volume.